core/core.py

```diff
@@ -1,73 +1,277 @@
+    :return: A JSON object containing the success state, which is True or False
     """
 
     data_dict = processing.json_to_dict(input_object)
 
     if data_dict["uid"] == "-1":
-        result_dict = database.add_component(data_dict)
+        result_dict = component_handler.add_component(data_dict)
         return processing.dict_to_json(result_dict)
     else:
-        result_dict = database.update_component(data_dict)
+        result_dict = component_handler.update_component(data_dict)
         return processing.dict_to_json(result_dict)
 
 
-def get_component(input_object: str) -> str:
+def delete_component(input_object: str) -> str:
     """
-    Receives a JSON object in the form defined under JSON_objects_defitnions.py for getting/viewing a component.
-    It returns another JSON object, sturctured as described in docu/JSON_objects_definitions.py
-    which is retreived from the component_handler.
+    Calls the delete_component method and returns whether successful or not in JSON Format
 
     :param input_object: JSON object containing the component uid
     :type input_object: str
-    :return: Returns a JSON object, sturctured as described in docu/JSON_objects_definitions.py
+    :return: String in JSON Format
     """
 
     data_dict = processing.json_to_dict(input_object)
-    component_dict = database.get_component(data_dict)
-    output_json = processing.dict_to_json(component_dict)
+    result_dict = component_handler.delete_component(data_dict)
+    output_json = processing.dict_to_json(result_dict)
 
     return output_json
 
 
-def delete_component(input_object: str) -> str:
-    """Calls the delete_component method and returns whether successful or not in JSON Format
+def get_process_list() -> str:
+    """
+    Calls the get_process_list method and converts the output to JSON
+
+    :return: A JSON formatted compontent list
+    """
+    process_list_dict = process_handler.get_process_list()
+    output_json = processing.dict_to_json(process_list_dict)
 
-    Args:
-        data (str): String in JSON Format
+    return output_json
 
-    Returns:
-        (str): String in JSON Format
 
-    :param input_object:
+def get_process(input_object: str) -> str:
     """
+    Receives a JSON object in the form defined under JSON_objects_defitnions.py for getting/viewing a process.
+    It returns another JSON object, sturctured as described in docu/JSON_objects_definitions.py
+    which is retreived from the process_handler.
 
+    :param input_object: JSON object containing the process uid
+    :type input_object: str
+    :return: Returns a JSON object, structured as described in docu/JSON_objects_definitions.py
+    """
     data_dict = processing.json_to_dict(input_object)
-    result_dict = database.delete_component(data_dict)
+    process_dict = process_handler.get_process(data_dict)
+    output_json = processing.dict_to_json(process_dict)
+
+    #TO DO Risk Calculation on output_json
```

Comment:
PEP 8: E265 block comment should start with '# '
```suggestion
    # TO DO Risk Calculation on output_json
```